src/edu/asu/plp/tool/prototype/model/PLPProject.java

```diff
@@ -7,6 +7,8 @@
 
 import javafx.beans.property.SimpleStringProperty;
 import javafx.beans.property.StringProperty;
+import javafx.stage.DirectoryChooser;
+import javafx.stage.FileChooser;
```

Comment:
PLPProject is a model class. It shouldn't have any GUI components or interactions (originating from this class), including FileChooser and DirectoryChooser.

These imports (and their usage) should be removed.